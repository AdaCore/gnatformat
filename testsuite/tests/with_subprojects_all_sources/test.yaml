description: Test that subproject are also formatted (except the ones marked as
  externally built).
driver: gnatformat
args: ["-P", "project_a/test_a.gpr", "--pipe", "-U"]
