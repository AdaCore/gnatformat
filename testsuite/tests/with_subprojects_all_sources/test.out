--  test_a.adb
with Test_A_Dep;

procedure Test_A
is
begin
   null;
end Test_A;

--  test_a_dep.ads
package Test_A_Dep is
   A : constant Integer := 1;
end Test_A_Dep;

--  test_a_unused_dep.ads
package Test_A_Unused_Dep is
   A : constant Integer := 1;
end Test_A_Unused_Dep;

--  test_b.adb
with Test_B_Dep;

procedure Test_B
is
begin
   null;
end Test_B;

--  test_b_dep.ads
package Test_B_Dep is
   A : constant Integer := 1;
end Test_B_Dep;

--  test_b_unused_dep.ads
package Test_B_Unused_Dep is
   A : constant Integer := 1;
end Test_B_Unused_Dep;

